examples/network_traffic_prediction/AirTrafficPrediction.ipynb

```diff
@@ -0,0 +1,735 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "9ba68c6f-d419-4f39-a3c4-340b256a83c6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nr_models import RgcnNRModel4TS\n",
+    "\n",
+    "# initialize the model with specific time series related arugments\n",
+    "ts_nfeat_names = {'airport':['inventory_amounts']}\n",
+    "ts_efeat_names = {('airport', 'demand', 'airport'): ['demands'], \\\n",
+    "                  ('airport', 'traffic', 'airport'): ['traffics']}\n",
+    "\n",
+    "# import a customized RGCN model for node time series regression\n",
+    "model = RgcnNRModel4TS(g=ml_data.g,\n",
+    "                       num_hid_layers=len(fanout),\n",
+    "                       node_feat_field=nfeats_4_modeling,\n",
+    "                       edge_feat_field=efeats_4_modeling,\n",
+    "                       edge_feat_mp_op='add',\n",
+    "                       target_ntype='airport',\n",
+    "                       ts_nfeat_names=ts_nfeat_names,\n",
+    "                       ts_efeat_names=ts_efeat_names,\n",
+    "                       hid_size=128,\n",
+    "                       ts_size=NUM_DEMAND_DAYS,\n",
+    "                       window_size=7)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "50883e00-ae86-4773-aa81-935fba9e72b6",
+   "metadata": {},
+   "source": [
+    "### 3. Setup a GraphStorm Evaluator\n",
+    "\n",
+    "To check the performance during model training, GraphStorm relies on a set of built-in `Evaluator`s for different tasks. Here we create a `GSgnnRegressionEvaluator` for the node time series regression task."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "12d5fca3-f026-460f-97b4-ed63427b1630",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# setup a regression evaluator for the trainer\n",
+    "from graphstorm.eval import GSgnnRegressionEvaluator\n",
+    "\n",
+    "evaluator = GSgnnRegressionEvaluator(eval_frequency=1000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5abb84ae-baff-4d2b-909d-743bb8b9b8b5",
+   "metadata": {},
+   "source": [
+    "### 4. Setup a Trainer and Training\n",
+    "\n",
+    "For training loop, GraphStorm has different `Trainer`s for specific tasks. GraphStorm's `GSgnnNodePredictionTrainer` is designed to orchestrate dataloaders, models, and evaluators, but cannot handle the sliding window method requirment. So here we extends the `GSgnnNodePredictionTrainer`, replacing its default `node_mini_batch_gnn_predict()` method with a customized method that supports sliding window for model training and validation, and autoregressive sliding window for inference. Users can find the details in the `nr_models.py` file.\n",
```

Comment:
"and autoregressive sliding window for inference." -> "and autoregressive sliding window for testing." ?

Reply:
Yes.